html_app/ui/FacsView.js

```diff
@@ -1405,51 +1490,51 @@ scb.ui.FacsView = function scb_ui_FacsView(gstate) {
-    	});
-		$(".scb_s_facs_selected").keypress(function(e){ return e.which != 13; });
-		
-		if(kind == 'analyze'){
-			$('.scb_s_facs_single_range_button').button();
-			$('.scb_s_facs_double_range_button').button();
-			if(state.facs.samples_finished && state.facs.selected_lane.bisector_gate_created) 
-				$('.scb_s_facs_double_range_button').button('disable');
-			
-		}
+        $("label[for='scb_facs_check']", '.scb_s_facs_view').attr('title', 'Single gate');
+        $("label[for='scb_facs_check2']", '.scb_s_facs_view').attr('title', 'Bisector gate');
+
+        $("label[class='scb_s_facs_label']", '.scb_s_facs_view').tooltip();
+
+        _.each($(".scb_s_experiment_step_button"), function (e) {
+            if (!$(e).hasClass('scb_s_experiment_step_visited'))
+                $(e).attr('title', 'To use this button, start a new ' + $(e).text() + ' Experiment.');
+            else $(e).removeAttr('title');
+        });
+        $(".scb_s_facs_selected").keypress(function (e) {
+            return e.which != 13;
+        });
+
+        if (kind == 'analyze') {
+            $('.scb_s_facs_single_range_button').button();
+            $('.scb_s_facs_double_range_button').button();
+            if (state.facs.samples_finished && state.facs.selected_lane.bisector_gate_created)
+                $('.scb_s_facs_double_range_button').button('disable');
+
+        }
 
         var elem = document.getElementById('slider');
-		window.mySwipe = Swipe(elem, {
-  			continuous: false,
-  			disableScroll: true,
-  			transitionEnd: function(index, element) {
-					$('.slider_dots li').attr('class','');
-					$($('.slider_dots li')[index]).attr('class','on');}
-		});
-		
-        
-		document.body.scrollTop = state.experiment.last_scroll;
-		
-		$('#main').css({
-				position:'absolute',
-				left: ($(window).width() - $('#main').outerWidth())/2,
-				top: 0
-			});
-		$(window).resize(function(){
-
-			$('#main').css({
-				position:'absolute',
-				left: ($(window).width() - $('#main').outerWidth())/2,
-				top: ($(window).height() - $('#main').outerHeight())/2
-			});
-
-		});
+            window.mySwipe = Swipe(elem, {
+                continuous: false,
+                disableScroll: true,
+                transitionEnd: function(index, element) {
+                  $('.slider_dots li').attr('class','');
+                  $($('.slider_dots li')[index]).attr('class','on');}
+                });
+        document.body.scrollTop = state.experiment.last_scroll;
+
+        $('#main').css({
+            position: 'absolute',
+            left: ($(window).width() - $('#main').outerWidth()) / 2,
+            top: 0
+        });
+        $(window).resize(function () {
+
+            $('#main').css({
+                position: 'absolute',
+                left: ($(window).width() - $('#main').outerWidth()) / 2,
+                top: ($(window).height() - $('#main').outerHeight()) / 2
+            });
+
+        });
     }
-}
\ No newline at end of file
+}
```

Comment:
This file is huge, is there any logical way to break it up into more than one, or does it make more sense as is?